FishjamClient/src/main/java/org/membraneframework/rtc/InternalMembraneRTC.kt

```diff
@@ -347,6 +351,7 @@ internal class InternalMembraneRTC(
                         }
                     }
                 }
+                canUpdateMetadata = true
```

Comment:
maybe we should use a mutex / coroutineScope instead (or other such mechanism)?
user can also do other things other than updating metadata so I'd prefer something that blocks other things completely